packages/plugin/src/theia.d.ts

```diff
@@ -2745,6 +2745,7 @@ export module '@theia/plugin' {
 
     /**
      * Options a virtual process terminal.
+     * @deprecated Use [ExtensionTerminalOptions](#ExtensionTerminalOptions) instead.
```

Comment:
We usually include `@since <version>` with our deprecations to help decide when to remove a deprecated feature.

Reply:
I looked through other deprecated annotations and it seems like they all give the since information without the `@since` annotation but rather a plain `since` string. So i stayed with that pattern. Let me now if you would rather have it with the annotation.